documentation/source/user_section/tutorials/non-template-registration/contrast-agnostic-registration.rst

```diff
@@ -0,0 +1,14 @@
+.. _contrast-agnostic-registration:
+
+Contrast agnostic registration using T1w and T2w spinal cord images
```

Comment:
```suggestion
Contrast-agnostic registration with deep learning
```

Given that T1w and T2w are only examples, i would not put them in the headings (too restrictive for the pristine eyes)